### PR TITLE
SEP-6: Deprecate wallet name and url

### PR DESCRIPTION
**Pull request by Ifropc**
See #1364

### CODE REVIEW

File: ecosystem/sep-0006.md [hunk 1]
```diff
@@ -1481,6 +1482,7 @@ If the information was malformed, or if the sender tried to update data that isn
 
 ## Changelog
 
+- `v3.2.0`: Deprecate the `wallet_name` and `wallet_url` parameters of `deposit` and `withdraw` requests ([#1393](https://github.com/stellar/stellar-protocol/pull/1393))
```
Comment (philipliu) [inline ecosystem/sep-0006.md:1484]:
```suggestion
- `v3.22.0`: Deprecate the `wallet_name` and `wallet_url` parameters of `deposit` and `withdraw` requests ([#1393](https://github.com/stellar/stellar-protocol/pull/1393))
```

**Review by JakeUrban — COMMENTED** [review]
Lets run prettier to keep the formatting consistent, but LGTM

**Comment by Ifropc** [comment]
I will run prettier in a separate PR

**Review by philipliu — APPROVED** [review]
LGTM

**Pull request merged**